shared/src/commonMain/kotlin/at/asitplus/wallet/app/common/ProvisioningService.kt

```diff
@@ -73,11 +74,13 @@ class ProvisioningService(
                         .also { dataStoreService.deletePreference(Configuration.DATASTORE_KEY_PROVISIONING_CONTEXT) }
                 }
         },
-        cryptoService = cryptoService,
-        holderAgent = holderAgent,
-        redirectUrl = redirectUrl,
-        clientId = clientId,
-        loadClientAttestationJwt = { clientAttestationJwt }
+        loadClientAttestationJwt = { clientAttestationJwt },
+        clientAttestationJwsService = DefaultJwsService(cryptoService),
+        oid4vciService = WalletService(clientId, redirectUrl, cryptoService),
+        storeCredential = { cred ->
+            runCatching { holderAgent.storeCredential(cred) }.onFailure { Napier.w("Could not store $cred", it)}
+        },
+        storeRefreshToken = {} // TODO store refresh tokens to refresh credentials later on
```

Comment:
I'm sure this is fine, but should is this already being worked on or at least tracked?